### ISSUE TITLE
Enable travis-buddy on GitHub repos

### ISSUE THREAD

**Issue by jayvdb** [coala/meta#114]
https://github.com/bluzi/travis-buddy provides a synopsis of the travis error on the pull request 

https://travisbuddy.com

**Comment by jayvdb** [comment]
I would suggest the following order

1. https://github.com/coala/documentation
2. https://github.com/coala/cEPs
3. https://github.com/coala/corobo
3. https://github.com/coala/coala-vs-code
3. https://github.com/coala/projects
3. https://github.com/coala/coala-html
4. https://github.com/coala/community
5. https://github.com/coala/coala-bears

And after each we check that it is working as expected before continuing to the next.

**Comment by manankalra** [comment]
There should be individual issues opened for this in the corresponding repos or just this issue should be referenced in the individual PRs?

**Comment by jayvdb** [comment]
No need, just use `Related to https://github.com/coala/meta/issues/114`

**Comment by palash25** [comment]
Can this issue have multiple assignee? I think we can complete it much faster if we divide individual repos amongst ourselves. If you don't mind @manankalra .

cc @jayvdb

**Comment by jayvdb** [comment]
Please read the issue again @palash25 .

**Comment by jayvdb** [comment]
Another option is to enable [travieso](https://github.com/wizeline/travieso)

maybe we try that after travis-buddy, if we dont like travis-buddy.

c.f. https://github.com/coala/coala/issues/4891

**Comment by manankalra** [comment]
travis isn't set up for cEPs; needed?

**Comment by jayvdb** [comment]
https://github.com/coala/documentation/pull/555#issuecomment-385757936 - a buddy error.  looks nice!

> travis isn't set up for cEPs; needed?

ok, skip that one for now.

**Comment by manankalra** [comment]
For corobo:
As per my knowledge, https://github.com/travis-ci/travis-ci/issues/5121 isn't solved yet. Also, the docs do not mention anything of this sort - https://docs.travis-ci.com/user/notifications/#Configuring-webhook-notifications.

Will have to omit `on_success: never` for now.

**Comment by jayvdb** [comment]
Before doing bears, here are some more to give us some experience with it

1. https://github.com/coala/coala-quickstart
2. https://github.com/coala/gci-leaders
3. https://github.com/coala/gh-board
4. https://github.com/coala/coala-emacs
5. https://github.com/coala/landing-frontend

**Comment by manankalra** [comment]
Should I open PRs for all of them?
There seems to be no ongoing contribution activities in any of the above repos (other than _gh-board_). Shouldn't we first test it on _coala-bears_ then?

**Comment by bluzi** [comment]
Hey guys, 

It looks like you've been using TravisBuddy for more than a month now, so I was just wondering if there's anything you think we can improve or add?

**Comment by jayvdb** [comment]
Interesting missing error at https://github.com/coala/corobo/pull/550#issuecomment-395577123

**Comment by manankalra** [comment]
Here, too. - https://github.com/coala/corobo/pull/552

**Comment by li-boxuan** [comment]
TravisBuddy copies too much log. 😕 That's a lot of network traffic, especially when we use API to fetch comments.

**Comment by jayvdb** [comment]
Probably https://github.com/bluzi/travis-buddy/issues/109 @bluzi 

Also some feature requests https://github.com/bluzi/travis-buddy/issues/created_by/jayvdb

@li-boxuan , can you link to some large comments. @bluzi may know how to fix them.

**Comment by li-boxuan** [comment]
This is a large comment: https://github.com/coala/community/pull/143#issuecomment-395946259

**Comment by jayvdb** [comment]
Any large comments in JS projects?

TravisBuddy doesnt have code for python specific projects yet.

**Comment by bluzi** [comment]
Thanks a lot guys! that's super helpful!

<hr />

> Interesting missing error at coala/corobo#550 (comment)

> Here, too. - coala/corobo#552

Thanks, will definitely look into it

<hr />

> Also some feature requests https://github.com/bluzi/travis-buddy/issues/created_by/jayvdb

Awesome, thank you!

<hr />

> This is a large comment: coala/community#143 (comment)

Definitely a bug, TravisBuddy should break the log using the `scripts` supplied in `.travis.yml`. Instead this comment contains the entire raw log.
Will look into it and get back to you.

**Comment by li-boxuan** [comment]
Not sure if this is also a 'large' comment (JS project) https://github.com/coala/gci-leaders/pull/149#issuecomment-386820394

**Comment by bluzi** [comment]
> Not sure if this is also a 'large' comment (JS project) coala/gci-leaders#149 (comment)

Nope. 
[As you can see in the build log](https://travis-ci.org/coala/gci-leaders/builds/375317464?utm_source=github_status&utm_medium=notification), the raw log is a lot bigger. TravisBuddy extracted the log of the executed command (`npm run build`), so that's by design.

**Comment by jayvdb** [comment]
@manankalra , any active repos left to enable this in?

**Comment by manankalra** [comment]
Yes. [coala](https://github.com/coala/coala) and [coala-bears](https://github.com/coala/coala-bears).
